index.bs

```diff
@@ -126,6 +132,12 @@ A [=crash report=]'s [=report/body=], represented in JavaScript by
   It represents whether the visual viewport that the crashed {{Document}} was hosted in was
   visible at all, or fully occluded.
 
+* Any other key/value data obtained by deserializing the memory buffer in [=user agent=]'s [=user
+  agent/crash report buffers=] map, identified by a crashing {{Document}}'s
+  {{CrashReportStorage}}'s [=CrashReportStorage/backing buffer ID=] member. This buffer is
+  deserialized as a JSON-formatted string, and its keys and values are appended to the
+  {{CrashReportBody}}.
```

Comment:
What happens if one of the keys is `stack`, `is_top_level`, or `page_visibility`? It might be better to nest this under a key.

Reply:
I think the `_api` suffix is unusual, but otherwise this makes sense.